kubelet/datadog_checks/kubelet/prometheus.py

```diff
@@ -264,11 +264,14 @@ def _sum_values_by_context(metric, uid_from_labels):
                 # metric.Clear()  # Ignore this metric message
         return seen
 
-    def _process_container_metric(self, type, metric_name, metric, scraper_config):
+    def _process_container_metric(self, type, metric_name, metric, scraper_config, labels=None):
         """
         Takes a simple metric about a container, reports it as a rate or gauge.
         If several series are found for a given container, values are summed before submission.
         """
+        if labels is None:
+            labels = []
```

Comment:
if you're doing this, why not `labels=[]` in the first place?

Reply:
yes 👍